db/pre-re/skill_db.yml

```diff
@@ -2106,6 +2106,7 @@ Body:
     SplashArea: 2
     ActiveInstance: 5
     Knockback: 2
+    CastCancel: false
```

Comment:
This seems like a useless change as the order shouldn't matter, does it?

Reply:
We could call it that, although keeping the file organized slightly improves the readability. This is a PR to prepare for the cooldown and cast time update.